app/main.py

```diff
@@ -22,11 +22,11 @@ def run_benchmark():
 results = run_benchmark()
 
 with cimetrics.upload.metrics() as m:
-    m.put("Throughput (tx/s)", results["throughput"])
+    m.put("Throughput (tx/s) ^", results["throughput"])
```

Comment:
The `^` should be removed and then the label trimmed for plotting, right?

Reply:
I don't know, I thought so at first, but then it seems like it's useful to have it to indicate that the expected direction is opposite to the default?

I'm almost seriously considering adding (📈👍) and (📉👍) next to each plot.

Reply:
Hm, doesn't the arrow color in the graph indicate this already?

Reply:
Ok, I'll hide it then!